src/System.Private.Xml/src/System/Xml/Xslt/XslTransform.cs

```diff
@@ -29,7 +28,7 @@ private XmlResolver _DocumentResolver
                 if (_isDocumentResolverSet)
                     return _documentResolver;
                 else
-                    return XsltConfigSection.CreateDefaultResolver();
```

Comment:
>CreateDefaultResolver [](start = 45, length = 21)

        internal static XmlResolver CreateDefaultResolver()
        {
            if (LocalAppContextSwitches.ProhibitDefaultUrlResolver)
                return null;
            else
                return new XmlUrlResolver();
        }
Shouldn't you be returning null then, not XmlNullResolver.Singleton (which is not null) ?